homeassistant/components/tellduslive/strings.json

```diff
@@ -1,6 +1,7 @@
 {
   "config": {
       "abort": {
+          "all_configured": "TelldusLive is already configured",
```

Comment:
Have the config-flow for tellduslive been updated with this?

Reply:
In all `tellduslive/.translations/[LANG].json`, this property is contained.

I've picked this one from the en.json file.

So I guess yes, but didn't look at it.

Reply:
AFAIK Ait is not used in the flow so this key should not be there.

Reply:
Okey, so I remove the key for all langs

Reply:
I've fixed your comment

Reply:
Can we resolve this @fredrike ?